sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/AnomalyDetection/AnomalyDetectionConfiguration.cs

```diff
@@ -79,7 +79,7 @@ internal AnomalyDetectionConfigurationPatch GetPatchModel()
                 DimensionGroupOverrideConfigurations = SeriesGroupDetectionConditions,
                 Name = Name,
                 SeriesOverrideConfigurations = SeriesDetectionConditions,
-                WholeMetricConfiguration = WholeSeriesDetectionConditions
+                //WholeMetricConfiguration = WholeSeriesDetectionConditions
```

Comment:
Service split an existing model in two similar types, so we'll need to make a conversion here.

Reply:
I imagine u already have issues for all of the changes that need to happen?

Reply:
They're all tracked in https://github.com/Azure/azure-sdk-for-net/issues/21177. This one is part of the "UpdateDetectionConfiguration" item given that tests won't pass if we don't fix this.